src/wiki-table-editor.jsx

```diff
@@ -38,34 +37,24 @@ class TableEditor extends React.Component {
     this.onRow = this.onRow.bind(this);
     this.onMoveRow = this.onMoveRow.bind(this);
     this.addRow = this.addRow.bind(this);
+    this.setCell = this.setCell.bind(this);
+    this.setNewCell = this.setNewCell.bind(this);
+    this.onFocus = this.onFocus.bind(this);
+    this.onBlur = this.onBlur.bind(this);
+    this.deleteButtonFormatter = this.deleteButtonFormatter.bind(this);
+    this.addNewButtonFormatter = this.addNewButtonFormatter.bind(this);
 
-    // Specify our custom editing behavior.
-    const editable = edit.edit({
```

Comment:
I stopped using `react-edit` since it swapped text `<input>`s in onclick. Instead, I'm just rendering `<input>`s all the time now.